versioned_docs/version-2.x/troubleshooting.md

```diff
@@ -0,0 +1,15 @@
+---
+id: troubleshooting
+title: Troubleshooting
+---
+
+
+When developing electron apps there are two processes: `main`, and `renderer`.  In this case `main` corresponds to `electron-starter.js` and its console is wherever you called `npm run dev` or `electron .` from. `renderer` corresponds to the React App - this is everything else. The react app's console is in the electron/browser window and can be seen by using dev tools to inspect the window.  When running `npm run dev`, it should open by default.
+
+### Potential Issues
+
```

Comment:
Might be a useful add-on: 
In case you want to find out where the error is coming from when running the app locally:
- Try reviewing logs in the terminal 
- Or use the inspect element in your browser by right-clicking and pressing inspect button on your web browser.

Reply:
Added! Thanks for the feedback!